src/StoreProvider.spec.lua

```diff
@@ -27,4 +27,33 @@ return function()
 			Roact.mount(element)
 		end).to.throw()
 	end)
+
+	it("should accept multiple children", function()
+		local store = Rodux.Store.new(function()
+			return 0
+		end)
+
+		local TestComponent = Roact.Component:extend("TestComponent")
+		local count = 0
+
+		function TestComponent:render()
+			count = count + 1
+		end
```

Comment:
I don't think this quite addresses the problem. Lucien's suggestion was that we make sure the children actually exist in the resulting tree. You could do this by parenting them to some object when you call mount (you could create a Folder and use that instead of parenting the tree to nil) and then inspecting the children via the Roblox engine API.